modules/base/rendering/renderablemodel.cpp

```diff
@@ -573,110 +682,226 @@ void RenderableModel::render(const RenderData& data, RendererTasks&) {
     // Formula from RenderableGlobe
     constexpr double tfov = 0.5773502691896257;
     constexpr int res = 2880;
-    const double maxDistance = res * boundingSphere() / tfov;
-
-    if (distanceToCamera < maxDistance) {
-        _program->activate();
-
-        _program->setUniform(_uniformCache.opacity, opacity());
-
-        // Model transform and view transform needs to be in double precision
-        const glm::dmat4 modelTransform =
-            glm::translate(glm::dmat4(1.0), data.modelTransform.translation) *
-            glm::dmat4(data.modelTransform.rotation) *
-            glm::scale(glm::dmat4(1.0), glm::dvec3(data.modelTransform.scale)) *
-            glm::scale(_modelTransform.value(), glm::dvec3(_modelScale));
-        const glm::dmat4 modelViewTransform = data.camera.combinedViewMatrix() *
-            modelTransform;
-
-        int nLightSources = 0;
-        _lightIntensitiesBuffer.resize(_lightSources.size());
-        _lightDirectionsViewSpaceBuffer.resize(_lightSources.size());
-        for (const std::unique_ptr<LightSource>& lightSource : _lightSources) {
-            if (!lightSource->isEnabled()) {
-                continue;
-            }
-            _lightIntensitiesBuffer[nLightSources] = lightSource->intensity();
-            _lightDirectionsViewSpaceBuffer[nLightSources] =
-                lightSource->directionViewSpace(data);
+    const double maxDistance =
+        res * boundingSphere() * glm::compMax(data.modelTransform.scale) / tfov;
```

Comment:
The bounding sphere function should already include the scale from the modeltransform?

Reply:
After discussion, I will add an issue and a todo comment